05/app.js

```diff
@@ -16,11 +16,14 @@ let valueLink = 0;
 
 const clickCounter = function (e) {
   e.preventDefault();
-  if (this.getAttribute("data-id") === "p1") {
+  if (e.target.tagName === "P" && e.target.getAttribute("data-id") === "p1") {
```

Comment:
Niestety to za mało. Należy zliczać wszystkie kliknięcia (czyli trzeba sprawdzać czy element istnieje, jeśli nie to go stworzyć). Jeszcze mała podpowiedź do tego:
```
const attr = e.target.getAttribute("data-id");
if(type stats.paragraphs[attr] === 'undefined') {
    stats.paragraphs[attr] = 1;
} else {
    stats.paragraphs[attr]++;
}
```

Reply:
Racja, poprawione. :)